services/web/webpack.config.js

```diff
@@ -27,7 +27,7 @@ const BUILD = process.env.NODE_ENV === 'production';
 
 const PARAMS = {
   BUILD,
-  ...require('./env'),
+  ...require('./serve/env').PUBLIC,
```

Comment:
there's no reason that the webpack build itself can't have all the ENV configs available to it as it's not considered "public" so maybe if we do the above and allow the env middleware to handle concept of public vs private then this can go away